src/tigerbeetle/cli.zig

```diff
@@ -105,6 +105,13 @@ const CliArgs = union(enum) {
         seed: ?u64 = null,
     },
 
+    // Internal: used to validate multiversion binaries.
+    multiversion: struct {
```

Comment:
I think ideally this would be a nested subcommand (but waiting for #2003 to land :wink:) so it would be:

```
tigerbeetle multiversion validate <path>
```

and we'd have a few others. Perhaps allowing exec'ing an internal version directly for debugging / etc purposes:
```
tigerbeetle multiversion execute <version> -- <args>
```